frequency.py

```diff
@@ -21,10 +44,33 @@ def get_top_n_words(word_list, n):
     punctuation
     n: the number of words to return
     returns: a list of n most frequently occurring words ordered from most
-    frequently to least frequentlyoccurring
+    frequently to least frequently occurring
     """
-    pass
+    histogram = {}
+    for word in word_list:
+        if word not in histogram:
+            histogram[word] = 1
+        else:
+            histogram[word] += 1
+    top_n_words = []
+    top_n_freqs = []
+    for i in range(n):
+        most_common = ''
+        most_times = 0
+        for key in histogram:
+            if key not in top_n_words:
+                num = histogram[key]
+                if num > most_times:
+                    most_times = num
+                    most_common = key
+        top_n_words.append(most_common)
+        top_n_freqs.append(most_times)
+    return top_n_words
 
 if __name__ == "__main__":
     print("Running WordFrequency Toolbox")
-    print(string.punctuation)
+    print(string.punctuation + string.whitespace)
+    word_list = get_word_list('pg32325.txt')
+    top_words = get_top_n_words(word_list, 100)
+    for word in top_words:
+        print(word)
```

Comment:
It might be cool to both print the word as well as the count of occurrences, but it works either way